### PR TITLE
Avoid exception swallowing

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

**Comment by sanjay990** [comment]
@electrum

File: src/main/java/com/facebook/presto/maven/ServiceDescriptorGenerator.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public void execute()
             pluginClasses = findImplementationsOf(Plugin.class, loader);
         }
         catch (Exception e) {
-            throw new MojoExecutionException(format("%n%nError scanning for classes implementing %s.", Plugin.class.getName()));
+            throw new MojoExecutionException(format("%n%nError scanning for classes implementing %s, %s", Plugin.class.getName(), e));
```
Comment (losipiuk) [inline src/main/java/com/facebook/presto/maven/ServiceDescriptorGenerator.java]: why not emboed `e` as `cause` in `MojoExecutionException`? Isn't it printed then?

Comment (sanjay990) [inline src/main/java/com/facebook/presto/maven/ServiceDescriptorGenerator.java]: Do you mean something like `throw new MojoExecutionException(format("%n%nError scanning for classes implementinghaha %s", Plugin.class.getName()), e);`. That works too.

File: pom.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
         <version>22</version>
     </parent>
 
-    <groupId>io.takari.maven.plugins</groupId>
+    <groupId>com.facebook.presto</groupId>
```
Comment (losipiuk) [inline pom.xml]: why this change?
Do we have `takari-maven-plugin` in `com.facebook.presto` maven group?
And how is that releated to this PR at all?

Comment (sanjay990) [inline pom.xml]: @electrum wanted to have this change. Its not related to this PR. I can create a new PR if required.

Comment (electrum) [inline pom.xml]: We need this so we can release a new version to Maven central. This project was originally written by @jvanzyl (who owns Takari) and I don't want to bother him every time we need to release :)

**Comment by electrum** [comment]
Merged, thanks!

File: pom.xml (hunk 2 above)
Comment (electrum) [inline pom.xml]: It's not related, I just asked him to throw it in here as a separate commit to make it easier. `takari-maven-plugin` comes from the Takari lifecycle defined in the base POM. It's fairly well documented now in this free book: http://takari.io/book/
